exports.createComment = async (req, res, next) => {
  try {
  } catch (error) {
    console.error('error occured in comment route - createComment :: \n', error);
    next(error);
  }
}

exports.deleteComment = async (req, res, next) => {
  try {
  } catch (error) {
    console.error('error occured in comment route - deleteComment :: \n', error);
    next(error);
  }
}
